types/ramda/index.d.ts

```diff
@@ -576,10 +576,10 @@ declare namespace R {
          * Takes a function and two values in its domain and returns true if the values map to the same value in the
          * codomain; false otherwise.
          */
-        eqBy<T>(fn: (a: T) => T, a: T, b: T): boolean;
-        eqBy<T>(fn: (a: T) => T, a: T): (b: T) => boolean;
-        eqBy<T>(fn: (a: T) => T): (a: T, b: T) => boolean;
-        eqBy<T>(fn: (a: T) => T): (a: T) => (b: T) => boolean;
+        eqBy<T, U = T>(fn: (a: T) => U, a: T, b: T): boolean;
+        eqBy<T, U = T>(fn: (a: T) => U, a: T): (b: T) => boolean;
+        eqBy<T, U = T>(fn: (a: T) => U): (a: T, b: T) => boolean;
+        eqBy<T, U = T>(fn: (a: T) => U): (a: T) => (b: T) => boolean;
```

Comment:
You didn't cause this, but the fourth overload is unreachable, since it has an identical parameter list to the third.

Reply:
That's right. I have removed the fourth overload and used the `CurriedFunction` type instead (as with similar cases in other Ramda functions).